Components/Typography/typography.css

```diff
@@ -0,0 +1,117 @@
+.typ-ex-grid {
+  display: grid;
+  grid-template-columns: max-content min-content;
+  grid-template-rows: repeat(auto, 50px);
+  align-items: center;
+  grid-gap: 30px;
+}
+
+.tag {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: max-content;
+  padding: 5px 7px;
+  height: 30px;
+  background-color: var(--yellow-color-light);
+  border: 2px solid var(--orange-color);
+  border-radius: 10px;
+}
+
+.tag p {
+  text-align: center;
+}
+
+p,
+a,
+h1,
+h2,
+h3,
+li,
+button {
+  font-family: "Open Sans", sans-serif;
+}
+
+.lg-title {
+  font-size: 3rem;
+  font-family: "Inter", sans-serif;
+  font-weight: 800;
+}
+.rg-title {
+  font-size: 2rem;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+}
+.md-title {
+  font-size: 1.5rem;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+}
+.sm-title {
+  font-size: 1.2rem;
+  font-family: "Inter", sans-serif;
+  font-weight: 800;
+}
+.xsm-title {
+  font-size: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 800;
+}
+
+.lg-booky-title {
+  font-size: 2.5rem;
+  font-family: "Libre Baskerville", serif;
+}
+.rg-booky-title {
+  font-size: 2rem;
+  font-family: "Libre Baskerville", serif;
+}
+
+.md-booky-title {
+  font-size: 1.5rem;
+  font-family: "Libre Baskerville", serif;
+}
+
+.sm-booky-title {
+  font-size: 1.2rem;
+  font-family: "Libre Baskerville", serif;
+}
+
+.rg-p {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1.4rem;
+}
+.sm-p {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1.2rem;
+}
+.xsm-p {
+  font-family: "Open Sans", sans-serif;
+  font-size: 12px;
+}
```

Comment:
common font family can be used. 

overall nice work! good to merge after changes

Reply:
they are different styles of titles not sure why its mandatory to use one font family when there are different use cases for each. Thanks for the compliment Gaurav.